data/mods/Magiclysm/recipes.json

```diff
@@ -11,5 +11,21 @@
     "byproducts": [ [ "stone_shell", 3 ] ],
     "qualities": [ { "id": "HAMMER", "level": 1 } ],
     "components": [ [ [ "egg_owlbear_rock", 1 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "glow_light",
+    "category": "CC_ELECTRONIC",
+    "subcategory": "CSC_ELECTRONIC_LIGHTING",
+    "skill_used": "spellcraft",
+    "difficulty": 0,
+    "time": "2 m",
+    "components": [
+      [ [ "mana_potion_lesser", 1 ] ],
+      [ [ "flask_glass", 1 ], [ "beaker", 1 ] ],
+      [ [ "glow_dust", 1 ] ],
```

Comment:
you know, i would make this a lot more expensive, because it seems really cheap to have a recipe like this at such a low difficulty autolearned that totally makes a spell obsolete.

Reply:
That's a good point. I figured it was kind of a dinky item giving off very little light but I hadn't considered it in competition with the spell as well. I can make it costlier.